src/js/components/FileInput/FileInput.js

```diff
@@ -85,6 +86,48 @@ const FileInput = forwardRef(
     const removeRef = useRef();
     const RemoveIcon = theme.fileInput.icons.remove;
 
+    const formatBytes = (size) => {
+      const units = ['B', 'KB', 'MB', 'GB', 'TB'];
+      const factor = 1000;
+      let index = 0;
+      let num = size;
+      while (num >= factor) {
```

Comment:
Probably should add a check for `index < units.length - 1` to be safe, however unlikely 1 PB+ is.